modules/policy/src/main/java/org/eclipse/xpanse/modules/policy/policyman/PolicyManager.java

```diff
@@ -44,16 +52,12 @@
 import org.springframework.web.client.RestClientException;
 
 /**
- * The service of policy-man.
+ * The service for managing policies.
  */
 @Slf4j
 @Component
 public class PolicyManager {
```

Comment:
this class has become too big. Would be good if we can split this into UserPolicyManager and ServicePolicyManager. We can move all common methods to another common class.

Reply:
Updated as suggested.